src/domain/community/user/user.interface.ts

```diff
@@ -36,6 +37,12 @@ export abstract class IUser extends INameable {
 
   agent?: IAgent;
 
+  @Field(() => [IUserPreference], {
```

Comment:
There is a field resolver for this, so it is not needed in the interface definition to also specify the field decorator